assets/js/app/llms-forms.js

```diff
@@ -378,12 +377,41 @@ LLMS.Forms = {
 			this.$states.html( '<option>&nbsp</option>' );
 			this.disable_field( this.$states );
 		} else {
+			var placeholder = this.get_state_placeholder( this.$states, country_code );
 			this.enable_field( this.$states );
 			this.$states.html( opts );
+			if ( '' !== placeholder ) {
+				this.set_state_placeholder( this.$states, placeholder );
+			}
 		}
 
 	},
 
+    /**
+	 * Get the state placeholder and/or {REGION}
+	 *
+	 * @since [version]
+	 *
+	 * @param {String} placeholder Currently selected state is the placeholder text.
+	 * @return {void}
+	 */
+	get_state_placeholder: function( holder, country_code ) {
+		var placeholder = holder.attr( 'placeholder' ).replace( '{REGION}', this.address_info[country_code].state );
```

Comment:
we need to put a check on `this.address_info[country_code].state` in the event that the specified country doesn't exist in the `address_info` object. With an additional fallback if the country *does* exist but the `state` property isn't defined...

Imagine an object like this:

```js
{
  abc: {
    state: '123',
  }
  xyz: {},
}
```
Results with different 

`this.address_info.abc.state` = 123 -- This is fine.
`this.address_info.xyz.state` = undefined -- This is also fine since passing undefined as the second parameter of `.replace()` has no adverse affects or errors
`this.address_info.efg.state` = Uncaught TypeError: this.address_info.efg is undefined -- This is not okay!

Reply:
That won't work because `country_code` is the string (not the object) and will always result in an error.

Other than that I think it's okay... I'd need to load it up and test it.

Passing "DEFAULT" will result in the placeholder string *printing* something like "Select a DEFAULT" though (we don't want that)